apps/Obsidian/install-64

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+enable_module fuse || exit 1
+
+wget -O /tmp/Obsidian.AppImage https://github.com/obsidianmd/obsidian-releases/releases/download/v0.9.17/Obsidian-0.9.17-arm64.AppImage || exit 1
+chmod +x /tmp/Obsidian.AppImage
+sudo mv -f /tmp/Obsidian.AppImage /opt || exit 1
```

Comment:
```suggestion
sudo mv -f /tmp/Obsidian.AppImage /opt || exit 1
sudo ln -sf /opt/Obsidian.AppImage /usr/bin/obsidian 
```

I don't think `/opt` is on `$PATH`.

Reply:
the description hasn't been written. telling the user to directly launch with /opt/Obsidian.AppImage is enough....
keep in mind this is a GUI app, and users should be launching with the .destop file (from their apps list) anyway

Reply:
@theofficialgman I find it more convenient to just type `obsidian`, if I'm launching from Terminal..

Reply:
> @theofficialgman I find it more convenient to just type `obsidian`, if I'm launching from Terminal..

Yes, but I don't want to remove the .AppImage suffix.
A symlink would be better.

Reply:
```
sudo mv -f /tmp/Obsidian.AppImage /opt || exit 1
```

Won't this line move `/tmp/Obsidian.AppImage` to the file `/opt`? Shouldn't it be:
```
sudo mv -f /tmp/Obsidian.AppImage /opt/ || exit 1
```